pkg/agent/sysadvisor/plugin/qosaware/resource/memory/plugin/memory_offloading.go

```diff
@@ -47,15 +48,19 @@ import (
 
 const (
 	TransparentMemoryOffloading = "transparent-memory-offloading"
+
+	MetricMemoryOffloading = "memory_offloading"
 )
 
 const (
 	InactiveProbe            = 0.1
 	OffloadingSizeScaleCoeff = 1.05
+	CacheMappedCoeff         = 2
 )
 
 const (
-	DummyTMOBlockFnName string = "dummy-tmo-block-fn"
+	DummyTMOBlockFnName           string = "dummy-tmo-block-fn"
+	TMOBlockFuncFromDynamicConfig string = "tmo-block-func-from-dynamic-config"
```

Comment:
Shouldn't the names of these two constants, blockFunc, be aligned？ for example: TMOBlockFuncFromDynamicConfig  -> FromDynamicConfigTMOBlockFnName

Reply:
ok